### PR TITLE
Hero tuneup

### PR DESCRIPTION
**Pull request by SylviaMakuch**
Please review the following for Hero: 

- Button for TestNet
- media queries

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "github.copilot-nightly"
+    ]
+}
```
Comment (ChristopherTrimboli) [inline .vscode/extensions.json]: don't need this file, can add `.vscode` to `.gitignore`

File: constants.ts [hunk 2]
```diff
@@ -64,9 +64,9 @@ export const socialLinks: Record<string, socialLink> = {
...
     },
-    // matrix: {
-    //     key: "matrix",
-    //     title: "Matrix",
-    //     url: "_"
-    // }
+    testnet: {
+        key: "testnet",
+        title: "TestNet",
+        url: "https://app.ownme.io/#/login"
```
Comment (ChristopherTrimboli) [inline constants.ts]: this URL should just be: `https://app.ownme.io/`

File: constants.ts [hunk 3]
```diff
@@ -64,9 +64,9 @@ export const socialLinks: Record<string, socialLink> = {
...
         url: "https://t.me/own_me_nft"
     },
-    // matrix: {
-    //     key: "matrix",
-    //     title: "Matrix",
-    //     url: "_"
-    // }
+    testnet: {
+        key: "testnet",
+        title: "TestNet",
```
Comment (ChristopherTrimboli) [inline constants.ts]: `Testnet`

File: sections/hero/TestNetButton.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+const Link = styled.a`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
+
+export default function TestNetButton() {
+    return (
+        <Button>
+            <Link href={socialLinks.testnet.url} target="_blank">Launch TestNet </Link>
```
Comment (ChristopherTrimboli) [inline sections/hero/TestNetButton.tsx]: blank space on end of: `Launch TestNet `

**Review by ChristopherTrimboli — CHANGES_REQUESTED** [review]
Looks great, few comments, please fix.

File: .vscode/extensions.json (hunk 1 above)
Comment (SylviaMakuch) [reply .vscode/extensions.json]: Done

File: constants.ts (hunk 2 above)
Comment (SylviaMakuch) [reply constants.ts]: done

File: constants.ts (hunk 3 above)
Comment (SylviaMakuch) [reply constants.ts]: typo fixed

File: sections/hero/TestNetButton.tsx (hunk 4 above)
Comment (SylviaMakuch) [reply sections/hero/TestNetButton.tsx]: Removed

File: .vscode/extensions.json (hunk 1 above)
Comment (ChristopherTrimboli) [reply .vscode/extensions.json]: did it wrong, just need to add: `.vscode` to file, not whole object... plus need to delete `.vscode` after it is stuck in Git now

Comment (ChristopherTrimboli) [reply .vscode/extensions.json]: fixed it

**Review by ChristopherTrimboli — APPROVED** [review]
looks great!

**Pull request merged**